templates/src/app/file/index.js

```diff
@@ -1,43 +1,37 @@
 var { Router } = require("express");
 var aws = require("aws-sdk");
-var cfsign = require("aws-cloudfront-sign");
 
-var router = Router()
+var router = Router();
 var s3 = new aws.S3();
 
-router.get("/presigned/:key", (req, res) => {
+router.get("/presigned", (req, res) => {
+  var bucketName = (req.query.bucket)?(req.query.bucket):process.env.AWS_S3_DEFAULT_BUCKET;
```

Comment:
This is a security risk, we shouldn't let a user choose which bucket to upload to.

Reply:
Removed the option. Thanks.